test/studies/filerator/walk-par.chpl

```diff
@@ -0,0 +1,13 @@
+use Filerator;
+
+config const startdir = ".";
+config const topdown = true;
+config const depth = max(int);
+config const dotfiles = false;
```

Comment:
While I could see bringing these into agreement, I'm also not 100% sure it's necessary given that the tests in question are in fact checking to see whether dotfiles will get printed out or not...  (which suggests that test tests won't work on a non-unix-based system, but we can cross that bridge when we get there).  Sound OK?